api/graphql/schema.graphql

```diff
@@ -1452,6 +1452,17 @@ input UpdatePoolInput {
   generalQuestions: UpdateGeneralQuestionsHasMany
 }
 
+input UpdatePublishedPoolInput @validator {
+  changeJustification: String @rename(attribute: "change_justification")
```

Comment:
Yeah, that makes sense. Updated in [make justification non nullable](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10348/commits/f36cbb50f62612028477c9bcc5f87f938ff50c03)

Reply:
That was actually causing problems and was really hard to test so I just have validation for it instead so we get a nice error message back rather then the  request being completely denied.

Reply:
Ah I see, well I am fine with it either way. But for reference sake, I believe this is usually how we write tests that fail on schema. It is a nuisance though with all the character escaping or caution needed 😆 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/e0411e77-c333-4508-a92c-e8f2f9ba4d75)

Reply:
Oh thanks, I know how to do it in the  future!  :stuck_out_tongue: .

I would like them more if we could write a nice message. It is hard to do much with that though :cry:

Reply:
Alright then 😆

Reply:
Okay, updated to be non nullable in [make field non-nullable](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10348/commits/928e258745cb4dcb38f43630cdf4b7fe206e1a75).

Though, I will try and push back once again to hopefully reconsider sometime in the future since using non-nullable fields as we do, while not explicitly incorrect, is not necessarily best practice and we should be handling null fields more gracefully than we are right :sweat_smile:

- [Using nullability in GraphQL](https://www.apollographql.com/blog/using-nullability-in-graphql)
- [When To Use GraphQL Non-Null Fields](https://medium.com/@calebmer/when-to-use-graphql-non-null-fields-4059337f6fc8)

Reply:
Okay, I'm interested in talking more about this and hearing your perspective. Even after reading those articles, I don't see anything wrong with how we're using nullability, especially when it comes to inputs.